extensions/iceberg/s3/src/test/java/io/deephaven/iceberg/util/IcebergParquetWriteInstructionsTest.java

```diff
@@ -40,11 +42,11 @@ void testSetVerifySchema() {
                 .verifySchema(true)
                 .build()
                 .verifySchema())
-                .isTrue();
+                .isEqualTo(Optional.of(true));
```

Comment:
In the case you are asserting against an optional, you can use the format `.hasValue(true)`.